graphql_compiler/compiler/expressions.py

```diff
@@ -811,6 +811,7 @@ def to_match(self):
         regular_operator_format = '(%(left)s %(operator)s %(right)s)'
         inverted_operator_format = '(%(right)s %(operator)s %(left)s)'  # noqa
         intersects_operator_format = '(%(operator)s(%(left)s, %(right)s).asList().size() > 0)'
+        negated_regular_operator_format = 'NOT (%(left)s %(operator)s %(right)s)'
```

Comment:
Note that all the other format strings wrap everything in parens. This is to avoid any chance of confusion about order of operations in the generated query. Let's make sure to wrap the entire expression here in parens as well, to continue that approach.

Reply:
Sounds good, will do